src/main/scala/org/kframework/minikore/interfaces/outer.scala

```diff
@@ -0,0 +1,67 @@
+package org.kframework.minikore.interfaces
+
+/**
+  * Created by lpena on 3/23/17.
+  */
+object outer {
+
+  import org.kframework.minikore.interfaces.{pattern => p}
+
+  type Attributes = Seq[p.Pattern]
+
+  trait HasAttributes {
```

Comment:
I'm worried about the name clash between `trait Attributes` and `type Attributes = Seq[p.Pattern]` being confusing. Thoughts?